src/core/manager-json.c

```diff
@@ -0,0 +1,2264 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <sys/mount.h>
+#include <sys/prctl.h>
+#include <sys/syslog.h>
+
+#include "automount.h"
+#include "build.h"
+#include "confidential-virt.h"
+#include "device.h"
+#include "exec-credential.h"
+#include "fileio.h"
+#include "json.h"
+#include "dbus-unit.h"
+#include "manager-json.h"
+#include "in-addr-prefix-util.h"
+#include "missing_ioprio.h"
+#include "parse-util.h"
+#include "rlimit-util.h"
+#include "process-util.h"
+#include "mount.h"
+#include "path.h"
+#include "scope.h"
+#include "service.h"
+#include "signal-util.h"
+#include "swap.h"
+#include "syslog-util.h"
+#include "timer.h"
+#include "version.h"
+#include "virt.h"
+#include "watchdog.h"
+
+#define JSON_BUILD_DUAL_TIMESTAMP(name, t) \
+        JSON_BUILD_PAIR_FINITE_USEC(name, t.realtime), \
+        JSON_BUILD_PAIR_FINITE_USEC(name "Monotonic", t.monotonic)
```

Comment:
hmm, i'd clean this up while we are at it, i.e. let's make the timestamps objects with two fields: "realtime" and "monotonic".

We should have done the same on D-Bus, but that ship is sailed. For JSON though we have the change to get this right